src/Packetery/Module/Checkout/Checkout.php

```diff
@@ -262,7 +263,12 @@ private function addAgeVerificationFee( ?Entity\Carrier $chosenCarrier, Carrier\
 	}
 
 	private function addCodSurchargeFee( Carrier\Options $carrierOptions, bool $isTaxable, ?string $maxTaxClass ): void {
```

Comment:
místo $this->wcAdapter->cartFeesApiAddFee() bych použil $cart->add_fee( $this->wpAdapter->__( 'COD surcharge', 'packeta' ), $surcharge, $taxable );  Ušetříme si pár řádků kódu.